docs/how-to/data/confidential-agent-config.toml.in

```diff
@@ -14,6 +14,7 @@ allowed = [
 "CreateContainerRequest",
 "CreateSandboxRequest",
 "DestroySandboxRequest",
+#"ExecProcessRequest",
```

Comment:
So I might have misunderstood, but those were the endpoints that Jakob didn't enable in the confidential demo. I just decided to make that explicit by adding them commented out as I noticed some other endpoints that we have been added since were missing. `ExecProcessRequest` makes sense to me as there is no point having a confidential container that the kubeadm can run any commands on. I'm not sure of the significance of `ReseedRandomDevRequest` , so was working on the assumption that Jakob knew better than me. If you've got a different list of 'sensitive' endpoints I'm very happy to update this!